_posts/2019-05-30-bitcoin-sov-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+In the same way that discovering the concepts of adding and subtracting quantities is more natural through comparing physical quantities (i.e. beads, as in an abacus), that does not necessarily prevent humans to skip that step and directly invent written numerals and ledgers.  Or, if for example, the standard within the abacus was the sexagesimal numeral system, that does not mean that is mandatory to go through an abacus stage of the decimal system in order to be able to adopt the written representation of the decimal numeral system. We must not confuse the supporting medium by which we historically experiment and discover a concept (i.e. beads or commodities) with the concept itself (arithmetics or Money).
+
+If something arises (bitcoin) that can fulfill the sole use case of a medium of exchange better than the incumbents (barter, salt, gold, fiat etc.), then that is where the demand can be found. Can this demand can be speculative? Yes, it is speculation on the possibility of the good becoming money in the future.
+
+Regarding completely new monetary goods and the fact that they don't have any other use than being a medium of exchange, Mises had the following concern (_Human Action_ - Chapter 17 section 4 – "The determination of purchasing power of money"):
+
+> A value judgment is, with reference to money, only possible if it can be based on appraisement. The acceptance of a new kind of money presupposes that the thing in question already has previous exchange value on account of the services it can render directly to consumption or production. Neither a buyer nor a seller could judge the value of a monetary unit if he had no information about its exchange value--its purchasing power--in the immediate past.
+
+So, at what price would a first exchange take place? That is the least of the problems. The owner of Bitcoin might arbitrarily ask for a price and buyers can accept it or bid at a lower price until they agree on the same price. At that point, the first exchange takes place. In the beginning, the price might be totally erratic, but as more market participants are interested, demand and supply builds and the price stabilizes. This continuous auction process is indeed how prices of goods have always been discovered and it is nothing special for money nor anything new that Bitcoin brought. The fact that many other speculators join in later just for the "lambos" without caring about Bitcoin's monetary utility is completely irrelevant in relation to Bitcoin's primary source of value. A different story is if those "heartless" speculators are good or bad for Bitcoin, but I'll leave that for another post.
```

Comment:
**Reword the first sentence to** "So, how can a new monetary good's price be discovered?"

**Reword the last sentence to** "A separate subject for another blog post is whether or not speculators and traders are beneficial for Bitcoin.